src/components/Banners/Implementations/WritersCohortBanner.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { Text } from "@chakra-ui/react"
+
+import DismissableBanner from "../DismissableBanner"
+import Link from "../../Link"
+
+interface IProps {
+  pathname: string
+}
+
+const WritersCohortBanner = ({ pathname }) => {
+  if (
+    typeof window !== `undefined` &&
+    (pathname.includes("contributing") || pathname.includes("community"))
+  ) {
```

Comment:
My idea was to completely remove the `window` dep in order to render it in both places. Or does it break?

```suggestion
  if (pathname.includes("contributing") || pathname.includes("community")) {
```